tests/test_skvbc_linearizability.py

```diff
@@ -80,8 +81,8 @@ class SkvbcChaosTest(unittest.TestCase):
     def test_healthy(self):
         """
         Run a bunch of concurrrent requests in batches and verify
-        linearizability. The system is healthy and stable and no faults are
-        intentionally generated.
+        linearizability. No crashed nodes, but some network packets
```

Comment:
Why change the healthy test to drop packets? This test was intended to serve as a baseline that our system is linearizable with no faults. 

Since these tests are somewhat expensive, I'm fine changing it, but I'd also change the name of the test to test_with_drops or something.

Reply:
+1, the intention is not very clear - I'll create a separate test case for this.